web/src/app/core/services/user.service.ts

```diff
@@ -75,4 +76,26 @@ export class UserService {
         switchMap(() => this.afs.doc<UserModel>(`userStats/${userId}`).valueChanges())
       );
   }
+
+  /**
+   * Update the following array inside users collection based upon the update flag
+   * @param userUid string
+   * @param projectUid string
+   * @param isUpdate flag if user uid has to remove or array in following array
+   */
+  public updateFollowings(userUid: string, projectUid: string, isUpdate: boolean): Observable<void> {
```

Comment:
The `userUid` is not needed to be provided, it will be the logged in user?

Reply:
Yes it should be logged in user